tests/airport.spec.js

```diff
@@ -1,18 +1,72 @@
 const Airport = require(`../src/Airport`);
 const { assertEquals } = require(`../testing-framework.js`);
-//const Plane = require(`../src/Plane`); - removed for loose coupling.
+const Plane = require(`../src/Plane`);
```

Comment:
In my airport spec I require the Plane class,. Looking back with new knowledge this is not loosely coupled and if I was doing this now I would use the Jasmine testing package and remove the coupling here for my tests.